### PR TITLE
Update the list of contribution in adopt a widget

### PR DESCRIPTION
**Pull request by Abhishek01039**

### CODE REVIEW

**Comment by Abhishek01039** [comment]
hello @johnpryan  
can you please review this PR?
Thanks

**Comment by Abhishek01039** [comment]
cc @sfshaza2

File: src/adoptawidget/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ similar in the future, so stay tuned!
 
 ## Contributions
 
-(Updated 10/11/2020)
+(Updated 12/02/2021)
```
Comment (sfshaza2) [inline src/adoptawidget/index.md:24]:
```suggestion
(Updated Feb 12, 2021)
```

File: src/adoptawidget/index.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ similar in the future, so stay tuned!
 - [redsolver](https://github.com/redsolver) contributed to the [Table docs](https://master-api.flutter.dev/flutter/widgets/Table-class.html) ([pull request](https://github.com/flutter/flutter/pull/69588))
 - [tsvillain](https://github.com/tsvillain) contributed to the [Tooltip](https://master-api.flutter.dev/flutter/material/Tooltip-class.html) ([pull request](https://github.com/flutter/flutter/pull/69518))
 - [Thesmader](https://github.com/Thesmader) contributed to the [SnackBar](https://master-api.flutter.dev/flutter/material/SnackBar-class.html) ([pull request](https://github.com/flutter/flutter/pull/69555))
+- [Abhishek Ghaskata](https://github.com/Abhishek01039) contributed to the [Hero](https://master-api.flutter.dev/flutter/widgets/Hero-class.html) ([pull request](https://github.com/flutter/flutter/pull/74415))
```
Comment (sfshaza2) [inline src/adoptawidget/index.md:40]:
```suggestion
- [Abhishek Ghaskata]({{site.github}}/Abhishek01039) contributed to the [Hero]({{site.master-api}}/flutter/widgets/Hero-class.html) ([pull request]({{site.github}}/flutter/flutter/pull/74415))
```

**Review by sfshaza2 — APPROVED** [review]
Oooh, somehow I never saw this page-they snuck it past me. I'd make lots of changes! For example, I'd have to check to repo to confirm whether this page appeared on 10/11 or 11/10. I think it was November 10th...  Also, we use "*" for bullets and macros for common URLs.

LGTM

**Comment by Abhishek01039** [comment]
Should I need to change anything @sfshaza2 ?

**Comment by sfshaza2** [comment]
No, that's fine!

**Comment by Abhishek01039** [comment]
Thanks @sfshaza2

**Pull request merged**